examples/ExchangeOracles/contracts/ExchangeMarket.aes

```diff
@@ -0,0 +1,58 @@
+contract ExchangeMarket =
+
+  record state = {
+    aePrice : int,
+    tokenPrice: int}
+
+  type query_type = string
+  type answer_type = int
+  type oracle_id = oracle(query_type, answer_type)
+  type query_id = oracle_query(query_type, answer_type)
+
+  public stateful function init(_aePrice: int, _tokenPrice: int) = 
+    require(_aePrice > 0 , "The price for the AE must be greater than zero ")
+    require(_tokenPrice > 0 , "The price for the token must be greater than zero")
+    {aePrice = _aePrice,
+      tokenPrice = _tokenPrice}
+    
+    
+  public function getAePrice() : int = state.aePrice
+  public function getTokenPrice() : int = state.tokenPrice
+
+  function createQuery(oracle: oracle_id, question: query_type, qfee: int, qttl: int, rttl: int) : query_id =
+    require(qfee > 0 , "Query fee must be greater than zero")
+    require(qttl > 0 , "Query TTL must be greater than zero")
+    require(rttl > 0 , "Relative TTL must be greater than zero")
+    Oracle.query(oracle, question, qfee, RelativeTTL(qttl), RelativeTTL(rttl))
+
+  function createAePriceQuery(oracle: oracle_id, qfee: int, qttl: int, rttl: int) : query_id =
+    require(qfee > 0 , "Query fee must be greater than zero")
+    require(qttl > 0 , "Query TTL must be greater than zero")
+    require(rttl > 0 , "Relative TTL must be greater than zero")
+    Oracle.query(oracle, "Give me the AE price in USD", qfee, RelativeTTL(qttl), RelativeTTL(rttl))
+
+  function createTokenPriceQuery(oracle: oracle_id, qfee: int, qttl: int, rttl: int) : query_id =
+    require(qfee > 0 , "Query fee must be greater than zero")
+    require(qttl > 0 , "Query TTL must be greater than zero")
+    require(rttl > 0 , "Relative TTL must be greater than zero")
+    Oracle.query(oracle, "Give me the Token price in USD", qfee, RelativeTTL(qttl), RelativeTTL(rttl))
+
+  function queryFee(oracle : oracle_id ) : int =
+    Oracle.query_fee(oracle)
+
+  public function getAnswer(oracle: oracle_id, query : query_id) : int =
+    switch(Oracle.get_answer(oracle, query))
+            None    => 0
```

Comment:
why not 2 spaces for indentation?

Reply:
also why multiple spaces after none? either way line up with the other => or just one space